hack/e2e.sh

```diff
@@ -19,6 +19,7 @@ if [[ "$FOCUS" == *"\[Fake\]"* ]]; then
     go generate ./...
     go run cmd/fakerp/main.go &
     trap 'return_id=$?; set +ex; kill $(lsof -t -i :8080); wait $(lsof -t -i :8080); exit $return_id' EXIT
+    while [[ "$(curl -s -o /dev/null -w '%{http_code}' localhost:8080)" == "000" ]]; do sleep 2; done
```

Comment:
This is making something bad worse :-(
Remove this hunk and fix https://github.com/openshift/openshift-azure/issues/1036 in a separate PR.

Reply:
we should be adding these retries in the client inside go instead of here?

Reply:
Sorry, ignore my comment above - it was late and I read this wrong.  This isn't ideal but it's fine for now.